### PR TITLE
doc: fix some more typos & grammar

### PR DESCRIPTION
**Pull request by DavidSint**
- Fix minor typos `knowning` --> `knowing` & `AOI` --> `API`
- Restructured line 38 to make more sense (unless I misunderstood it, in which case, please correct me)
- Change `nodejs` to `Node.js` for consistency

### CODE REVIEW

**Review by helio-frota — APPROVED** [review]
LGTM

File: auth.md [hunk 1]
```diff
@@ -23,26 +23,26 @@
...
+* A pure Backend API that never deals with a fronted should never be concern of redirecting or dealing with end users, they would require an access/API token or assume the proxy/gateway in front is already handling this and not require token at all. https://github.com/ibm-cloud-security/appid-video-tutorials/blob/master/02b-simple-node-backend-app/app.js
 
 * The browser/client should never have access to access token.
 
 * The Authorization Server will interact with the user and once is authenticated it will return to the browser with a grant code, which in turn can be used by the Web App request an access token. With this access token the WebApp can access a Backend API for a resource.
 
 * Use the refresh token whenever possible, this avoids re-authentication.
 
-* Do not use OAUTH2 implicit grant, is preferred to use Authorization code workflow.
+* Do not use OAUTH2 implicit grant, instead the Authorization code workflow is preferred.
```
Comment (helio-frota) [inline auth.md:38]: that seems more clear to me :+1:

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@DavidSint seems like there is a conflict. Could you rebase?

**Comment by wtrocki** [comment]
@DavidSint Hi. Are you happy for me to cherry-pick your commit to master in order to not lose this changes and do not leave PR hanging? +1 is enough to get it sorted.

**Comment by lholmquist** [comment]
closed via https://github.com/nodeshift/nodejs-reference-architecture/commit/cfca93ee62947fcf476eaa907df7b8326a433ab8

**Pull request closed without merging**